### DISCUSSION TITLE
Backpack\CalendarOperation drag & drop does not work properly

### DISCUSSION THREAD

**Discussion by Sladous** [Laravel-Backpack/community-forum#1155] [Q&A (Help)] [answered]
Hi,
I purchased Backpack\CalendarOperation and am experiencing several issues with it:
1) Not even on the demo
https://demo.backpackforlaravel.com/admin/meeting/calendar
saving mouse changes does not work correctly.
If the all-day event is moved back one day, the system returns "Event updated successfully", but after reloading the page, the event is moved back one day.

2) It is not allowed to comment:
// use \Backpack\CRUD\app\Http\Controllers\Operations\ShowOperation;
Because the context menu doesn't work otherwise.
I solve:
 $this->crud->removeButton('show');

3) I don't know how to set Monday as the first day of the week.
Could this be related to point 1?

**Comment by karandatwani92** [comment]
Hey @Sladous 

Sorry for inconvience. I have escalated the [issue](https://github.com/Laravel-Backpack/calendar-operation/issues/34) for the fix.
I will update here once fixed. Thanks for reporting.

**Comment by karandatwani92** [comment]
Hey @Sladous 

Can you try `composer update`
- 1 and 2 are fixed and merged.
- For 3rd, I will update you later on this.

**Reply by Sladous** [reply]
1 nad 2 are OK. Thx.
3. I am waiting for:     
$this->crud->setOperationSetting('week_number_calculation', 1);
:-)

**Reply by karandatwani92** [reply, marked as answer]
@Sladous To set `Monday` as the first day of the week, you define the following:

```php
public function setupCalendarOperation()
{
    CRUD::setOperationSetting('javascript-configuration', [
        // ... FullCalendar options
        // https://fullcalendar.io/docs
        'firstDay' => 1, // by default it's 0 - sunday
    ]);
}
```

https://fullcalendar.io/docs/firstDay

**Comment by Sladous** [comment]
4. I have one last question. The calendar ignores the filters I have set in listoperation. How to fix it? Thx.
